src/connect.js

```diff
@@ -20,7 +20,7 @@
  * @return {Function}
  */
 function connect(params) {
-  // write code here
+  return (extra) => Object.assign({}, params, extra);
```

Comment:
Good, also you can write like this
```suggestion
  return (extraParams) => ({ ...params, ... extraParams });
```